upsertGitHubTag/deployment/index.js

```diff
@@ -267,9 +267,10 @@ function logPayloadToS3(body, deliveryId) {
   // If bucket name is not null (had to put this for the integration test)
   if (process.env.BUCKET_NAME) {
     const uploadDate = new Date();
-    const bucketPath = `${uploadDate.getFullYear()}
-          -${uploadDate.getMonth() + 1}
-          -${uploadDate.getDate()}/${deliveryId}`; //formats path to YYYY-MM-DD/deliveryid
+    const bucketPath =
+      `${uploadDate.getFullYear()}` +
+      `-${uploadDate.getMonth() + 1}` +
```

Comment:
It would be nice if the length were always the same, e.g., `2024-05-02` instead of `2024-5-2`. It was like this, but I only noticed it now.

Reply:
Screenshot updated :)